### PR TITLE
Include new Firebase admin syntax+Readme update

### PR DESCRIPTION
**Pull request by siege-nnn**
Hi,
I tested the proposed pull.

Please verify and commit for others benefit

### CODE REVIEW

**Comment by siege-nnn** [comment]
I signed it!

**Comment by siege-nnn** [comment]
I signed it!

**Comment by siege-nnn** [comment]
I signed it!

**Comment by siege-nnn** [comment]
I signed it!

File: src/firebase-import.js [hunk 1]
```diff
@@ -42,11 +43,13 @@ var argv = require('optimist')
   .argv;
 
 function main() {
-  firebase.initializeApp({
-    databaseURL: argv.database_url,
-    serviceAccount: argv.service_account,
+console.log(argv.service_account)
```
Comment (mikelehen) [inline src/firebase-import.js]: Was this just for debugging? Can it be removed?

File: src/firebase-import.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ var firebase = require('firebase'),
     fs = require('fs'),
     JSONStream = require('JSONStream'),
     util = require('util');
+    admin = require("firebase-admin");
```
Comment (mikelehen) [inline src/firebase-import.js]: Does this need to be added to the package.json dependencies?

**Pull request closed without merging**